homeassistant/components/buienradar/camera.py

```diff
@@ -0,0 +1,176 @@
+"""Provide animated GIF loops of Buienradar imagery."""
+import asyncio
+import logging
+from datetime import datetime, timedelta
+from typing import Optional
+
+import aiohttp
+import voluptuous as vol
+
+from homeassistant.components.camera import PLATFORM_SCHEMA, Camera
+from homeassistant.const import CONF_ID, CONF_NAME
+
+from homeassistant.helpers import config_validation as cv
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+
+from homeassistant.util import dt as dt_util
+from homeassistant.util.async_ import run_coroutine_threadsafe
+
+CONF_DIMENSION = 'dimension'
+CONF_DELTA = 'delta'
+
+RADAR_MAP_URL_TEMPLATE = ('https://api.buienradar.nl/image/1.0/'
```

Comment:
Can we add the constant to the existing PyPI package?

Reply:
Last comment is regarding this. Would be great if we could just do something like `url = radar_url(x, y)`

Reply:
And import `radar_url` from the dependency.

Reply:
Contributed this to the library and awaiting the result of a pull request upstream. Will update code of the this PR + the buienradar sensor afterwards